libraries/tools/kotlin-symbol-processing-api/src/org/jetbrains/kotlin/ksp/utils.kt

```diff
@@ -74,7 +68,14 @@ fun KSTypeAlias.findActualType(): KSClassDeclaration {
  */
 fun KSDeclaration.getVisibility(): Visibility {
     return when {
-        this.isLocal() -> Visibility.LOCAL
+        this.modifiers.contains(Modifier.OVERRIDE) -> {
+            when (this) {
+                is KSFunctionDeclaration -> this.findOverridee()?.getVisibility()
+                is KSPropertyDeclaration -> this.findOverridee()?.getVisibility()
+                else -> null
+            } ?: Visibility.PUBLIC
```

Comment:
`else -> Visibility.PUBLIC`

Reply:
other 2 branches in when statement can also return null, therefore it's better to put elvis statement out of when statement

Reply:
Ah, you are right. Didn't see the `?`.